test/hermes/intl/to-locale-lowercase.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// RUN: %hermes -O -target=HBC %s | %FileCheck --match-full-lines %s
+// REQUIRES: intl
+
+print("get locale lowercase test");
+// CHECK-LABEL: get locale lowercase test
+
+print(`${'TEST'.toLocaleLowerCase()}`);
```

Comment:
These don't need to be in template literals (should apply to all of them)
```suggestion
print('TEST'.toLocaleLowerCase());
```